src/components/offer/offer.jsx

```diff
@@ -0,0 +1,348 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Offer = (props) => {
+
+  const {offer} = props;
+
+  const {
+    price,
+    name,
+    premium,
+    rating,
+    features,
+    gallery,
+    insides,
+  } = offer;
+
+  const {
+    entire,
+    bedrooms,
+    adults,
+  } = features;
+
+  return (
+    <div className="page">
+      <header className="header">
+        <div className="container">
+          <div className="header__wrapper">
+            <div className="header__left">
+              <a className="header__logo-link" href="main.html">
+                <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41" />
+              </a>
+            </div>
+            <nav className="header__nav">
+              <ul className="header__nav-list">
+                <li className="header__nav-item user">
+                  <a className="header__nav-link header__nav-link--profile" href="#">
+                    <div className="header__avatar-wrapper user__avatar-wrapper">
+                    </div>
+                    <span className="header__user-name user__name">Oliver.conner@gmail.com</span>
+                  </a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </header>
+
+      <main className="page__main page__main--property">
+        <section className="property">
+          <div className="property__gallery-container container">
+            <div className="property__gallery">
+
+              {
+                gallery.map((photoSrc, index) => {
+                  return (
+                    <div className="property__image-wrapper" key={index}>
```

Comment:
```suggestion
                    <div className="property__image-wrapper" key={photoSrc}>
```
если фотографии уникальные